nservicebus/testing/index_content_testing_[6,).partial.md

```diff
@@ -29,4 +26,6 @@ snippet: SampleBehavior
 
 The behavior can be tested similar to a message handler or a Saga by using a testable representation of the context:
 
-snippet: BehaviorTest
\ No newline at end of file
+snippet: BehaviorTest
+
+Testable representations are provided for all pipeline behaviour contexts, see [the pipeline documentation](/nservicebus/pipeline/) for further details.
```

Comment:
Should be "behavior", not "behaviour".

Comma should be changed to a period, "see" should be "See"